cinema/migrations/0003_alter_movie_actors_alter_movie_genres.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1 on 2024-01-03 20:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0002_actor_cinemahall_genre_movie_actors_movie_genres'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='actor', to='cinema.actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='movie', to='cinema.genre'),
+        ),
+    ]
```

Comment:
It is a good practice in Django projects to include one migration per app per pull request with all changes made to the models in this app to not overload the `migrations` folder. Follow this approach in the next pull requests, please.